FabricTestExample/src/Test654.js

```diff
@@ -5,8 +5,6 @@ import { createNativeStackNavigator } from '@react-navigation/native-stack';
 
 const Stack = createNativeStackNavigator();
 
-I18nManager.forceRTL(true);
-
```

Comment:
Revert that change and merge branch with main

Reply:
https://github.com/software-mansion/react-native-screens/pull/2131/commits/1786094ba9e5195ecef8a693cbaa759eaa6d8ce5